javagen/src/main/java/com/azure/autorest/mapper/ClientMethodMapper.java

```diff
@@ -283,6 +283,7 @@ private List<ClientMethod> createClientMethods(Operation operation, boolean isPr
                     .requiredNullableParameterExpressions(requiredParameterExpressions)
                     .validateExpressions(validateExpressions)
                     .methodTransformationDetails(methodTransformationDetails)
+                    .methodVisibilityInWrapperClient(isProtocolMethod && operation.isGenerateProtocolApi() != null && operation.isGenerateProtocolApi() == Boolean.FALSE ? JavaVisibility.PackagePrivate : JavaVisibility.Public)
```

Comment:
I think `operation.isGenerateProtocolApi() == Boolean.FALSE` covers `!= null`?

Reply:
Yes, should be covered, just adding `!=null` to make sure we consider the case where `isGenerateProtocolApi` not set is covered. I can remove that.

Reply:
Yes, please remove that.